src/general/disallowanonymous/src_test.rego

```diff
@@ -0,0 +1,248 @@
+package k8sdisallowanonymous
+
+test_blank_subject_clusterrolebinding {
+    input := {"review": clusterrolebinding([{}], "role-2"), "parameters": {"allowedRoles": ["role-1"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_non_anonymous_clusterrolebinding {
+    input := {"review": clusterrolebinding([{"name": "user-1", "kind": "User"}], "role-2"), "parameters": {"allowedRoles": ["role-1"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_anonymous_user_clusterrolebinding {
+    input := {"review": clusterrolebinding([{"name": "system:anonymous", "kind": "User"}], "role-2"), "parameters": {"allowedRoles": ["role-1"]}}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_allowed_role_anonymous_user_clusterrolebinding {
+    input := {"review": clusterrolebinding([{"name": "system:anonymous", "kind": "User"}], "role-2"), "parameters": {"allowedRoles": ["role-2"]}}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_multiple_subjects_anonymous_user_clusterrolebinding {
```

Comment:
Do we have a test with just `system:authenticated`?
```rego
input := {"review": clusterrolebinding([{"name": "system:authenticated", "kind": "Group"}], "role-2"), "parameters": {"allowedRoles": ["role-1"]}}
count(results) == 0
```

Reply:
Thanks @ritazh. I added the test.